css-ecommerce-page/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link rel="stylesheet" href="styles.css">
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container font">
+    <div class="row">
+      <div class="column-full text-center">
+        <h1>New Arrivals</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-fifth">
+        <img src="images/dress1.jpeg" alt="Free People Dress">
+        <div class="name-size">
+          <p>Free People</p>
+          <p>Size 6</p>
+        </div>
```

Comment:
Instead of making this a `div` element with `p` child elements, I would make this a `p` element with `span` child elements